ripuc.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+import scrapy
+
+
+class RipucSpider(scrapy.Spider):
+    name = 'ripuc'
+    allowed_domains = ['ripuc.org']
+    start_urls = ['http://www.ripuc.org/eventsactions/docket.html']
+
+    def parse(self, response):
+        table = response.xpath('//td[@class="normal"]/table')
+        number_of_rows = int(float(table.xpath('count(tr)').extract_first()))
```

Comment:
Do we really need to count the records and then loop through them ?
can we do like this:
```
for doket in docket_table.xpath(...):
    pass
```
As we are using this index below, we can rename "i" to something better.